.circleci/config.yml

```diff
@@ -101,5 +101,14 @@ jobs:
       - checkout
       - setup_remote_docker:
           docker_layer_caching: true
-      - run: make release-publish
-
+      - run:
+          name: 'Set custom environment variables'
+          command: |
+            echo 'export GIT_TAG=$(git describe --tags | tr -d "v")' >> $BASH_ENV
+            source $BASH_ENV
+      - run:
+          name: 'Generate release and publish'
+          command: make release-publish
+      - run:
+          name: 'Generate homebrew-core pull request'
+          command: ./scripts/brew-formula.sh
```

Comment:
Goreleaser also supports post-build hooks so this could be part of the process above.  It should be part / called via `make release-publish` if you don't want to use the post-build hooks.  https://goreleaser.com/build/

Reply:
I'll take a look at this today.

Reply:
🗒️: To close the loop, it turns out GoReleaser's `post` hook within `build` doesn't contain the archived assets so we can't run the script via the `build.hooks.post` hook. And although there's a global top-level `before.hooks`, there is no global top-level `after.hooks`. It would be ideal to have the global `after` hook as well as a `post` hook within the `archives` configuration.  ¯\\_(ツ)_/¯